flocker/control/test/test_script.py

```diff
@@ -95,7 +108,10 @@ def test_no_immediate_stop(self):
         """
         script = ControlScript()
         options = ControlOptions()
-        options.parseOptions([b"--data-path", self.mktemp()])
+        options.parseOptions([
+            b"--data-path", self.mktemp(),
+            b"--certificates-directory", self.certificate_path.path
+        ])
```

Comment:
These various `ControlOptions.parseOptions` calls with the same `b"--certificates-directory", self.certificate_path.path` elements in every call would be a good thing to factor out. to reduce the repetition.